### DISCUSSION TITLE
Need help to make script for windows11

### DISCUSSION THREAD

**Discussion by Ahmad1769** [koreader/koreader#11378] [Q&A] [answered]
The backend koreader-base (https://github.com/koreader/koreader-base/) has 'make TARGET=win32' as an option, and the shell script file (https://github.com/koreader/koreader/blob/master/kodev) has win32 as a target for the 3rd party tools (./kodev fetch-thirdparty) and the emulator (./kodev build), but not for the actual release (./kodev release). 

I noticed the makefile for the koreader repository has these lines in them:

# releases do not contain tests and misc data

IS_RELEASE := $(if $(or $(EMULATE_READER),$(WIN32)),,1)

IS_RELEASE := $(if $(or $(IS_RELEASE),$(APPIMAGE),$(DEBIAN),$(MACOS)),1,)

I think that makes it so WIN32 doesn't have a release build, right? I tried changing them and compiling with mingw64/msys2 on Windows, but couldn't even get past the luajit build. I'd love to hear your thoughts! Thank you!

**Comment by Frenzie** [comment, marked as answer]
In short, see #7653. There's no such thing as a Windows build.
